### PR TITLE
CHANGES: prepare for 1.8 release

### PR DESCRIPTION
**Pull request by jluebbe**

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -3,20 +3,54 @@ Release 1.8 (development)
...
+* Ignore ``meta.<label>`` sections in the manifest.
+  The ``meta.<label>`` sections are intended to provide a forwards-compatible
+  way to add data to the manifest which is not interpreted by RAUC in any way.
+  Currently, they are just ignored when reading a manifest. In future releases,
+  they will be accessible via rauc info, the D-Bus API and in hooks/handlers.
 
 .. rubric:: Bug fixes
 
+* Avoid retrying on HTTP 404 errors during streaming.
+* Improve error handing during loop device block size configuration. (by Ahmad
```
Comment (OnkelUlla) [inline CHANGES:35]:
```suggestion
* Improve error handling during loop device block size configuration. (by Ahmad
```

**Review by OnkelUlla — CHANGES_REQUESTED** [review]
Only a small typo, otherwise it looks good to me.

File: CHANGES [hunk 2]
```diff
@@ -3,20 +3,54 @@ Release 1.8 (development)
...
+  Ludovico de Nittis)
+* Add support for a RAUC data-directory on a shared partition.
+  See the :ref:`documentation for details <data-directory>`.
+* Allow setting a passphrase for encrypted PEM files via the environment
+  (``RAUC_KEY_PASSPHRASE``). (by Marc Kleine-Budde)
+* Ignore ``meta.<label>`` sections in the manifest.
+  The ``meta.<label>`` sections are intended to provide a forwards-compatible
+  way to add data to the manifest which is not interpreted by RAUC in any way.
+  Currently, they are just ignored when reading a manifest. In future releases,
+  they will be accessible via rauc info, the D-Bus API and in hooks/handlers.
```
Comment (ejoerns) [inline CHANGES:30]:
```suggestion
  they will be accessible via ``rauc info``, the D-Bus API and in hooks/handlers.
```

File: CHANGES [hunk 3]
```diff
@@ -3,20 +3,54 @@ Release 1.8 (development)
...
+  See the :ref:`documentation for details <sec-incremental-updates>`.
+* Add a slot type which provides atomic bootloader updates for SoCs (like the
+  Rockchip RK3568) which search for a valid image at multiple fixed offsets. (by
+  Matthias Fend)
+  See the :ref:`documentation for details <sec-raw-partition-fallback>`.
+* Add a configuration option for additional arguments to pass to ``casync
+  extract``. (by Ludovico de Nittis)
+* Add initial support for desync (an alternative casync implementation). (by
+  Ludovico de Nittis)
+* Add support for a RAUC data-directory on a shared partition.
```
Comment (ejoerns) [inline CHANGES:22]: Maybe note that this directory is used for the central slot status unless otherwise configured?

File: CHANGES [hunk 4]
```diff
@@ -3,20 +3,54 @@ Release 1.8 (development)
...
 .. rubric:: Testing
 
+* Refactor the statistics code to make it useful for testing as well.
 
 .. rubric:: Code
 
+* Log error messages from CURL for failed streaming requests.
+* Add doctype to D-Bus XML specification. (by Morgan Bengtsson)
 
 .. rubric:: Documentation
```
Comment (ejoerns) [inline CHANGES:48]: * Document how to load and store the GRUB environment from a shared partition

File: CHANGES [hunk 5]
```diff
@@ -3,20 +3,54 @@ Release 1.8 (development)
...
+  Currently, they are just ignored when reading a manifest. In future releases,
+  they will be accessible via rauc info, the D-Bus API and in hooks/handlers.
 
 .. rubric:: Bug fixes
 
+* Avoid retrying on HTTP 404 errors during streaming.
+* Improve error handing during loop device block size configuration. (by Ahmad
+  Fatoum)
 
 .. rubric:: Testing
```
Comment (ejoerns) [inline CHANGES:39]: * Replace ubuntu 21.10 test container with 22.04

**Review by ejoerns — APPROVED** [review]

**Pull request merged**